### PR TITLE
Add ManagedServerSecurityAlertPolicy and ManagedDatabaseSecurityAlert…

### PR DESCRIPTION
**Pull request by talhers**
…Policies

### Latest improvements:
<i>MSFT employees can try out our new experience at <b>[OpenAPI Hub](https://aka.ms/openapiportal) </b> - one location for using our validation tools and finding your workflow. 
</i><br>
### Contribution checklist:
- [x] I have reviewed the [documentation](https://github.com/Azure/azure-rest-api-specs#basics) for the workflow.
- [x] [Validation tools](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/swagger-checklist.md#validation-tools-for-swagger-checklist) were run on swagger spec(s) and have all been fixed in this PR.
- [x] The [OpenAPI Hub](https://aka.ms/openapiportal) was used for checking validation status and next steps.

### CODE REVIEW

**Comment by talhers** [comment]
@yaakoviyun @yoavfr

**Comment by yaakoviyun** [comment]
@ayeletshpigelman

**Comment by talhers** [comment]
> @talhers Have the new APIs been through an API review?

These APIs are similar to the regular server and database APIs:

https://github.com/Azure/azure-rest-api-specs/blob/master/specification/sql/resource-manager/Microsoft.Sql/stable/2014-04-01/databaseSecurityAlertPolicies.json

https://github.com/Azure/azure-rest-api-specs/blob/master/specification/sql/resource-manager/Microsoft.Sql/preview/2017-03-01-preview/serverSecurityAlertPolicies.json

The difference is that they apply to managed instance and managed database (except for that they are identical)

**Review by ravbhatnagar — COMMENTED** [review]
@talhers - some comments. please take a look.

File: specification/sql/resource-manager/Microsoft.Sql/preview/2017-03-01-preview/ManagedDatabaseSecurityAlertPolicies.json [hunk 1]
```diff
@@ -0,0 +1,337 @@
...
+    "https"
+  ],
+  "consumes": [
+    "application/json"
+  ],
+  "produces": [
+    "application/json"
+  ],
+  "paths": {
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Sql/managedInstances/{managedInstanceName}/databases/{databaseName}/securityAlertPolicies/{securityAlertPolicyName}": {
```
[the remark this answers is not in the document]
Comment (talhers) [reply specification/sql/resource-manager/Microsoft.Sql/preview/2017-03-01-preview/ManagedDatabaseSecurityAlertPolicies.json]: We will open a work item for this

File: specification/sql/resource-manager/Microsoft.Sql/preview/2017-03-01-preview/ManagedDatabaseSecurityAlertPolicies.json [hunk 2]
```diff
@@ -0,0 +1,337 @@
...
+  "schemes": [
+    "https"
+  ],
+  "consumes": [
+    "application/json"
+  ],
+  "produces": [
+    "application/json"
+  ],
+  "paths": {
```
[the remark this answers is not in the document]
Comment (talhers) [reply specification/sql/resource-manager/Microsoft.Sql/preview/2017-03-01-preview/ManagedDatabaseSecurityAlertPolicies.json]: After thinking about it we don't think that a patch is needed in our customer scenarios

File: specification/sql/resource-manager/Microsoft.Sql/preview/2017-03-01-preview/ManagedDatabaseSecurityAlertPolicies.json [hunk 3]
```diff
@@ -0,0 +1,337 @@
...
+        },
+        "emailAccountAdmins": {
+          "description": "Specifies that the alert is sent to the account administrators.",
+          "type": "boolean"
+        },
+        "storageEndpoint": {
+          "description": "Specifies the blob storage endpoint (e.g. https://MyAccount.blob.core.windows.net). This blob storage will hold all Threat Detection audit logs.",
+          "type": "string"
+        },
+        "storageAccountAccessKey": {
```
[the remark this answers is not in the document]
Comment (talhers) [reply specification/sql/resource-manager/Microsoft.Sql/preview/2017-03-01-preview/ManagedDatabaseSecurityAlertPolicies.json]: We will open a work item for this

File: specification/sql/resource-manager/Microsoft.Sql/preview/2017-03-01-preview/ManagedServerSecurityAlertPolicy.json [hunk 4]
```diff
@@ -0,0 +1,313 @@
...
+  "schemes": [
+    "https"
+  ],
+  "consumes": [
+    "application/json"
+  ],
+  "produces": [
+    "application/json"
+  ],
+  "paths": {
```
[the remark this answers is not in the document]
Comment (talhers) [reply specification/sql/resource-manager/Microsoft.Sql/preview/2017-03-01-preview/ManagedServerSecurityAlertPolicy.json]: We will open a work item for this

**Comment by ravbhatnagar** [comment]
Couple of pending issues Tracked here https://github.com/Azure/azure-rest-api-specs/issues/4910
Signing off from ARM side.

**Comment by ravbhatnagar** [comment]
#4910

**Review by praries880 — APPROVED** [review]

**Pull request merged**